README.md

```diff
@@ -1,8 +1,15 @@
 # kind-image
 Kind-node image for the PowerPC architecture.
 
-Images are hosted at https://quay.io/repository/powercloud/kind-node?tab=tags and can be used as follows:
+Before spinning up the cluster, one may need to install the kind binary with the git patch applied to the repository 
+to use ppc64le specific images while setting up the cluster.
+
+Follow the installation guide from [here](./docs/install.md)
```

Comment:
I think this needs to be linked to above statement or why should I follow this guide?

Reply:
Added context.